pyproject.toml

```diff
@@ -57,6 +57,10 @@ help = "Generate test cases (do this once before running test)"
 cmd = "pytest"
 help = "Run tests"
 
+[tool.poe.tasks.test-coverage]
```

Comment:
I don't think we need this, we can just have `pytest --cov betterproto` in the `tox` config.

Reply:
This has now been removed and Tox no longer uses Poe, as it seems to use the Poetry Virtualenv no matter what, which defeats the point of this fix.